client/client.go

```diff
@@ -293,6 +297,7 @@ func (c *client) getOrCreateGRPCConn(addr string) (*grpc.ClientConn, error) {
 	defer c.connMu.Unlock()
 	if old, ok := c.connMu.clientConns[addr]; ok {
 		cc.Close()
+		log.Warn("use old connection", zap.String("target", cc.Target()), zap.String("state", cc.GetState().String()))
```

Comment:
Maybe Info or Debug? It is expected when there multiple goroutines trying to create connection at the same time?